<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternPluginMaker</class>
 <widget class="QMainWindow" name="ExternPluginMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExternPluginMaker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Plugin Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="PluginName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Release Date</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Copyright</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Contributors</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Plugin Type</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Sub Type</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="Version"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="ReleaseDate"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="Copyright"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="Contributors"/>
       </item>
       <item row="9" column="1">
        <widget class="QPlainTextEdit" name="Desc"/>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="PluginType">
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data Processor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IO Driver</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>File Transfer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tool</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="SubType">
         <item>
          <property name="text">
           <string>Data Processor Char Encoding</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data Processor Term Emulation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data Processor Highlighter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data Processor Logger</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data Processor Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Linux's .so</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Window's .dll</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Mac's .so</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="Linux_SO"/>
          </item>
          <item>
           <widget class="QPushButton" name="SelectLinuxFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="Windows_DLL"/>
          </item>
          <item>
           <widget class="QPushButton" name="SelectWindowsFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="Mac_SO"/>
          </item>
          <item>
           <widget class="QPushButton" name="SelectMacFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>API Version</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="APIVersion"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Install Filename</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="InstallFilename"/>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>(Does not include extension)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Raspberry PI 64</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Raspberry PI 32</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="RPI64_SO"/>
          </item>
          <item>
           <widget class="QPushButton" name="SelectRPI64File">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="RPI32_SO"/>
          </item>
          <item>
           <widget class="QPushButton" name="SelectRPI32File">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="BuiltBttn">
         <property name="text">
          <string>Build</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="LoadBttn">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SaveBttn">
         <property name="text">
          <string>Save...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCommand">
    <property name="title">
     <string>&amp;Command</string>
    </property>
    <addaction name="actionBuild"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommand"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>&amp;Build</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>PluginName</tabstop>
  <tabstop>Version</tabstop>
  <tabstop>APIVersion</tabstop>
  <tabstop>ReleaseDate</tabstop>
  <tabstop>Copyright</tabstop>
  <tabstop>Contributors</tabstop>
  <tabstop>PluginType</tabstop>
  <tabstop>SubType</tabstop>
  <tabstop>Desc</tabstop>
  <tabstop>Linux_SO</tabstop>
  <tabstop>SelectLinuxFile</tabstop>
  <tabstop>Windows_DLL</tabstop>
  <tabstop>SelectWindowsFile</tabstop>
  <tabstop>Mac_SO</tabstop>
  <tabstop>SelectMacFile</tabstop>
  <tabstop>BuiltBttn</tabstop>
  <tabstop>LoadBttn</tabstop>
  <tabstop>SaveBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
